gaphor/ui/elementeditor.py

```diff
@@ -34,6 +35,32 @@
 new_builder = new_resource_builder("gaphor.ui", "elementeditor")
 
 
+LANGUAGES = [
+    (None, gettext("System")),
+    ("ca", gettext("Catalan")),
+    ("zh_Hans", gettext("Chinese (Simpl.)")),
+    ("hr", gettext("Croatian")),
+    ("cs", gettext("Czech")),
+    ("nl", gettext("Dutch")),
+    ("C", gettext("English")),
+    ("de", gettext("German")),
+    ("fi", gettext("Finnish")),
+    ("fr", gettext("French")),
+    ("gl", gettext("Galician")),
+    ("hu", gettext("Hungarian")),
+    ("it", gettext("Italian")),
+    ("ja", gettext("Japanese")),
+    ("ko", gettext("Korean")),
+    ("pl", gettext("Polish")),
+    ("pt_BR", gettext("Portuguese (Brazil)")),
+    ("ru", gettext("Russian")),
+    ("sl", gettext("Slovenian")),
+    ("es", gettext("Spanish")),
+    ("sv", gettext("Swedish")),
+    ("tr", gettext("Turkish")),
```

Comment:
We could also get those language names from the Babel database. That would safe us some translation work.

Only, how do we know what the actual language is?

Reply:
You mean what the system language is set to?

I like the idea to get the names from Babel. Could we parse the name of each folder in gaphor/locale to get the list of supported languages? Then `babel.Locale.get_language_name()` should be able to translate the name, I think.